src/App/Components/DisplayArticle/readArticle.js

```diff
@@ -0,0 +1,128 @@
+/* eslint-disable react/destructuring-assignment */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import propTypes from 'prop-types';
+// import Popular from '../Popular/popular';
```

Comment:
@MCFrank16 the commented code is still here

Reply:
this is for the popular in case the PO doesn't like the new look of the popular